davealex/quick-credit
# #165587483 Design Platform Help Page

#### What does this PR do?
This PR adds an help.html page to the Project
#### Description of Task to be completed?
On this page, a user can access a curated list of FAQs and helpful answers and is able to filter through the list with a search bar
#### How should this be manually tested?
Navigate to the help.html page and ensure that the page is properly rendered when served on mobile devices
Also, ensures that the search bar filters the list as expected
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#165587483
#### Screenshots (if appropriate)
#### Questions:
